swagger.yaml

```diff
@@ -3778,6 +3791,22 @@ definitions:
       console_url:
         type: string
 
+  disk-config-params:
+    type: object
+    required:
+      - 'id'
+    properties:
+      id:
+        type: string
+      role:
+        $ref: "#/definitions/disk-role"
+
+  disk-role:
+    type: string
+    enum:
+      - 'none'
```

Comment:
isn't it confusing? the default is probably empty string, so unset should be empty string as well no?

Reply:
This is per the API design from @avishayt . roles will be extended in the future, so there must be a placeholder for "Unknown" (or not specified) role

Reply:
Why not default to "none"?

Reply:
It's strange to have default here, when use call for this api he will not specify all the disks, just the relevant one for him. so how do you expect to set a `none` as default?

Reply:
We would specify all the disks, and eventually each disk would have its own role

Reply:
from this swagger it's not clear where do you store it.